INFO/pom.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project
+        xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+        xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>THBINGEN</groupId>
+    <artifactId>InformatikKlausur</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <kotlin.version>1.1.2</kotlin.version>
+        <main.class>src.main.kotlin</main.class>
```

Comment:
Hier habe ich HelloWorldFileKt eingetragen. wird neuer Fehler angezeigt "Exception in thread "main" java.lang.NoClassDefFoundError..."